odysseus/HeraclesQuest.ts

```diff
@@ -187,7 +187,7 @@ export type HeraclesQuestTask = HeraclesQuestElement & ({
     item: RegistryValue;
     nbt?: JsonObject;
     amount?: number;
-    collection_type?: 'automatic' | 'manual' | 'consume';
+    collection?: 'AUTOMATIC' | 'MANUAL' | 'CONSUME';
```

Comment:
Why switch lowercase to uppercase? I believe @ThatGravyBoat's codec is case insensitive for enums

Reply:
if you edit the quest in Heracles and then save it, it'll spit out upper-case - so I wanted there to be less diff.

Reply:
Alright, lgtm